src/wp-includes/cache.php

```diff
@@ -131,6 +131,34 @@ function wp_cache_set_multiple( array $data, $group = '', $expire = 0 ) {
 
 	return $wp_object_cache->set_multiple( $data, $group, $expire );
 }
+/**
+ * Removes all cache items in a group.
+ *
+ * @since 6.0.0
+ *
+ * @see WP_Object_Cache::flush_group()
+ * @global WP_Object_Cache $wp_object_cache Object cache global instance.
+ *
+ * @param string|array $group name(s) of group to remove from cache.
+ *
+ * @return bool True on success, false on failure group not found.
+ */
+function wp_cache_flush_group( $group ) {
+	global $wp_object_cache;
+
+	// if group is an array loop and call each key in the array
+	if ( is_array( $group ) ) {
+		array_map( 'wp_cache_flush_group', array_values( $group ) );
+
+		return true;
```

Comment:
This is a bit ambiguous. Should we return an array of booleans, if an array is given? @spacedmonkey?